GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -0,0 +1,174 @@
+
+```shell-script
+$ git clone https://github.com/kiegroup/kogito-examples
+cd kogito-examples/process-business-rules-quarkus
+mvn clean package
+```
+Now create a Dockerfile inside the `kogito-examples/process-business-rules-quarkus`. With the following content:
+
+```Dockerfile
+FROM quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest
+
+COPY target/*-runner.jar $KOGITO_HOME/bin    
+COPY target/lib $KOGITO_HOME/bin/lib
+```
+
+```shell-script
+$ podman build --tag quay.io/<yournamespace>/process-business-rules-quarkus:latest .
+```
+It’ll build your image with the kogito service running on it.
+
+Now let’s push the above built image to a public container registry.
+
+`$ podman push quay.io/<yournamespace>/process-business-rules-quarkus:latest`
+
+## Install the Operator
+
+First we would need to enable the OLM in our Minikube cluster. For that, just run:
+
+`$ minikube addons enable olm`
+
+To launch the OLM console locally, clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repository and from the root of the project run: `$ make run-console-local`.
+ 
+This will run the operatorhub console on http://localhost:9000 
+
+**Note**: You will need to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where `kogito-operator` and all the dependent operator(s) will run
+
+`$ kubectl create ns kogito`
+
+Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`.  
+Select the Kogito Operator by Red Hat and install it with defaults options, choose the namespace `kogito` which was created for this purpose
+
+You can see the pods by:
+
+`$ watch kubectl get pod -n kogito`
+
+Wait until all pods are in running state (It is installing kogito-operator and all the dependent operator(s) for kogito)
+
+Alternatively, you can install the `Kogito Operator` from [here](https://operatorhub.io/operator/kogito-operator)
+
+## Run Kogito Service with Kogito Operator
+
+Now we’ll create a `KogitoRuntime` object with the image we built and pushed earlier.
+
+```yaml
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoRuntime
+metadata:
+  name: process-business-rules-quarkus
+spec:
+  replicas: 1
+  image:
+    registry: quay.io
+    name: process-business-rules-quarkus
+    tag: latest
+    namespace: <your-quay-namespace>
+```
+Create a `myapp.yml` file with the above content.
+
+Just create the above object with
+
+`$ kubectl create -f myapp.yml -n kogito`
+
+You can see your runtime object by 
+
+`$ kubectl get KogitoRuntime -n kogito`
+
+Now expose the service on NodePort so it’s easily accessible.
+
+`$ kubectl expose deployment process-business-rules-quarkus -n <operator-namespace>  --type=NodePort --name=process-business-rules-quarkus`
```

Comment:
also need to mention that the ingress addon should be enabled, I don't think it is enabled by default.